docker/pg_tests/scripts/tests/several_delta_backups_test.sh

```diff
@@ -9,6 +9,7 @@ cat ${COMMON_CONFIG} >> ${TMP_CONFIG}
 /tmp/scripts/wrap_config_file.sh ${TMP_CONFIG}
 
 /usr/lib/postgresql/10/bin/initdb ${PGDATA}
+ln -sf /dev/null "${PGDATA}/postgresql.auto.conf"
```

Comment:
I would add some comment that indicates our intention to test the correct symlink handling.

Reply:
I added some context.